website-custom-views/src/content/api-reference/custom-view-config.mdx

```diff
@@ -474,7 +468,7 @@ Supported values are: `SMALL`, `LARGE`.
 
 ## `locators`
 
-Since Custom Views are rendered in the context of native Merchant Center applications, you need to set in which specific views the Custom View should be rendered.
+Since Custom Views are rendered in the context of built-in Merchant Center applications, you need to set in which specific views the Custom View should be rendered.
```

Comment:
```suggestion
Since Custom Views are rendered in the context of built-in Merchant Center applications, you need to define the specific views where the Custom View should be rendered.
```